doc/source/ray-core/scheduling/placement-group.rst

```diff
@@ -562,16 +403,159 @@ You can remove a placement group at any time to free its allocated resources.
       ray::PlacementGroup removed_placement_group = ray::GetPlacementGroup(placement_group.GetID());
       assert(removed_placement_group.GetState(), ray::PlacementGroupState::REMOVED);
 
-Named Placement Groups
-----------------------
+.. _ray-placement-group-observability-ref:
+
+Observe and Debug Placement Groups
+----------------------------------
+
+Ray provides several useful tools to inspect the placement group states and resource usage.
+
+- **Ray Status** is a CLI tool to see the resource usage and the scheduling resource requirement of the placement groups.
+- **Ray Dashboard** is a UI tool to inspect placement group states.
+- **Ray State API** is a CLI to inspect placement group states.
+
+.. tabbed:: Ray Status
+
+  The CLI command ``ray status`` provides the autoscaling status of the cluster. 
+  It provides the "resource demands" from unscheduled placement groups as well as the resource reservation status.
+
+  SANG-TODO images
+
+.. tabbed:: Dashboard
+
+  :ref:`The dashboard job view <dash-jobs-view>` provides the placement group table that displays the scheduling state and metadata of the placement group.
+
+  .. note::
+
+    Ray dashboard is only available when Ray is installed with ``pip install "ray[default]"``.
+
+.. tabbed:: Ray State API
+
+  :ref:`Ray state API <state-api-overview-ref>` is a CLI tool to inspect the state of Ray resources (tasks, actors, placement groups, etc.). 
+
+  ``ray list placement-groups`` provides the metadata and the scheduling state of the placement group.
+  ``ray list placement-groups --detail`` provides stats and scheduling state in a greater detail.
+
+  .. note::
+
+    State API is only available when Ray is installed with ``pip install "ray[default]"``
+
+Inspect Placement Group Scheduling State
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+From the above tools, you can see the state of the placement group. The definition of states are specified in the following files.
+
+- `High level state <https://github.com/ray-project/ray/blob/03a9d2166988b16b7cbf51dac0e6e586455b28d8/src/ray/protobuf/gcs.proto#L579>`_
+- `Details <https://github.com/ray-project/ray/blob/03a9d2166988b16b7cbf51dac0e6e586455b28d8/src/ray/protobuf/gcs.proto#L524>`_
+
+.. _pgroup-strategy:
+
+[Advanced] Placment Strategy
```

Comment:
is it really advanced?

Reply:
Hmm yeah since it is one of the core features of the API, we should just consider as non advanced. fixed.